pkg/yurtctl/constants/constants.go

```diff
@@ -224,7 +219,8 @@ spec:
       - name: yurt-controller-manager
         image: {{.image}}
         command:
-        - yurt-controller-manager	
+        - yurt-controller-manager
+        - --controllers=-servicetopologycontroller,-poolcoordinatorcertmanager,*
```

Comment:
These two controllers do not run by default?

Reply:
these two controllers depend on the yurt-app-manager component, but in e2e tests, yurt-app-manager component is not deployed, so disable these two controllers.

Reply:
OK